documentation/docs/13-page-options.md

```diff
@@ -70,6 +70,15 @@ For _pages_, we skirt around this problem by writing `foo/index.html` instead of
 
 Note that this will disable client-side routing for any navigation from this page, regardless of whether the router is already active.
 
+#### Troubleshooting
+
+If you encounter an error like 'Route ... is marked as prerenderable, but was not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't actually prerendered.
```

Comment:
```suggestion
If you encounter an error like 'The following routes were marked as prerenderable, but were not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't actually prerendered.
```